samcli/local/init/__init__.py

```diff
@@ -13,29 +13,53 @@
 _init_path = os.path.dirname(__file__)
 _templates = os.path.join(_init_path, 'templates')
 
-RUNTIME_TEMPLATE_MAPPING = {
-    "python3.7": os.path.join(_templates, "cookiecutter-aws-sam-hello-python"),
-    "python3.6": os.path.join(_templates, "cookiecutter-aws-sam-hello-python"),
-    "python2.7": os.path.join(_templates, "cookiecutter-aws-sam-hello-python"),
-    "python": os.path.join(_templates, "cookiecutter-aws-sam-hello-python"),
-    "ruby2.5": os.path.join(_templates, "cookiecutter-aws-sam-hello-ruby"),
-    "nodejs6.10": os.path.join(_templates, "cookiecutter-aws-sam-hello-nodejs6"),
-    "nodejs8.10": os.path.join(_templates, "cookiecutter-aws-sam-hello-nodejs"),
-    "nodejs": os.path.join(_templates, "cookiecutter-aws-sam-hello-nodejs"),
-    "dotnetcore2.0": os.path.join(_templates, "cookiecutter-aws-sam-hello-dotnet"),
-    "dotnetcore2.1": os.path.join(_templates, "cookiecutter-aws-sam-hello-dotnet"),
-    "dotnetcore1.0": os.path.join(_templates, "cookiecutter-aws-sam-hello-dotnet"),
-    "dotnetcore": os.path.join(_templates, "cookiecutter-aws-sam-hello-dotnet"),
-    "dotnet": os.path.join(_templates, "cookiecutter-aws-sam-hello-dotnet"),
-    "go1.x": os.path.join(_templates, "cookiecutter-aws-sam-hello-golang"),
-    "go": os.path.join(_templates, "cookiecutter-aws-sam-hello-golang"),
-    "java8": os.path.join(_templates, "cookiecutter-aws-sam-hello-java"),
-    "java": os.path.join(_templates, "cookiecutter-aws-sam-hello-java")
+RUNTIME_DEP_TEMPLATE_MAPPING = {
```

Comment:
There is a lot of mappings to keep updated all together. Can we condense these into one?

Reply:
Will see how to condense this.